### PR TITLE
Detection improvements

### PR DESCRIPTION
**Pull request by DaimonPl**
https://github.com/piwik/device-detector/issues/5547
https://github.com/piwik/device-detector/issues/5543

### CODE REVIEW

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -81,9 +81,15 @@ HTC:
...
       device: 'tablet'
-      model: '$1'
+      model: 'P715a'
+    - regex: 'HTC[ _\-]Flyer Build'
+      device: 'tablet'
+      model: 'Flyer'
+    - regex: 'HTC[ _\-]Flyer[ _\-]([\w]{1,5})'
+      device: 'tablet'
+      model: 'Flyer $1'
     - regex: 'HTC[ _\-]One[ _\-]max[);/ ]'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: Aren't the HTC Flyer devices more like Phablets? Guess all of them have 7'' ?

File: Tests/fixtures/tablet.yml [hunk 2]
```diff
@@ -9595,10 +9675,250 @@
...
+    engine: WebKit
+    engine_version: "534.30"
+  device:
+    type: tablet
+    brand: LE
+    model: Yoga Tablet 10
+  os_family: Android
+  browser_family: Android Browser
+-
+  user_agent: Mozilla/5.0 (Linux; U; Android 4.2.2;pl-pl; Lenovo B8000-F/JDQ39) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.2.2 Mobile Safari/534.30
```
Comment (sgiehl) [inline Tests/fixtures/tablet.yml]: Guess that test can be removed, as it's quite the same as two before (the one you changed the detected model)

**Review by sgiehl — CHANGES_REQUESTED** [review]
Looks good. Left two small inline comments. Please check those. Afterwards we can merge.

File: Tests/fixtures/tablet.yml (hunk 2 above)
Comment (DaimonPl) [inline Tests/fixtures/tablet.yml]: They are almost the same but still some differences (some rules are expecting "Build" sometimes so thought it's worth leaving those)
Lenovo B8000-F Build/JDQ39
Lenovo B8000-H/JDQ39
Lenovo B8000-F/JDQ39

File: regexes/device/mobiles.yml (hunk 1 above)
Comment (DaimonPl) [inline regexes/device/mobiles.yml]: thought phablets are 5.5–6.99 inches, also for exemple here http://www.gsmarena.com/htc_flyer-3829.php it's marked as tablet

let me know what's piwik approach when something is phablet and when something is tablet :)

File: Tests/fixtures/tablet.yml (hunk 2 above)
Comment (sgiehl) [inline Tests/fixtures/tablet.yml]: Ah, didn't see that. It's fine then.

File: regexes/device/mobiles.yml (hunk 1 above)
Comment (sgiehl) [inline regexes/device/mobiles.yml]: Hm. thats a good question ;-) Guess the range of 5.5–6.99 inches is fine. Wasn't sure about counting 7'' as phablet or not. But we can leave it like it is.

**Review by sgiehl — APPROVED** [review]

**Pull request merged**